dask_jobqueue/oar.py

```diff
@@ -123,6 +163,11 @@ class OARCluster(JobQueueCluster):
         Deprecated: use ``job_extra_directives`` instead. This parameter will be removed in a future version.
     job_extra_directives : list
         List of other OAR options, for example `-t besteffort`. Each option will be prepended with the #OAR prefix.
+    oar_mem_core_property_name : str
+        The memory per core property name of your OAR cluster (usually named `memcore` or `mem_core`).
+        Existing properties can be listed by `oarnodes` command.
+        Note that the memory per core property might not exist on your cluster.
```

Comment:
Ajouter 'In such a case, do not specify a value for oar_mem_core_property_name parameter' ?